janusgraph-dist/src/assembly/static/cassandra/conf/cassandra.yaml

```diff
@@ -632,7 +832,7 @@ request_scheduler: org.apache.cassandra.scheduler.NoScheduler
 #
 # The passwords used in these options must match the passwords used when generating
 # the keystore and truststore.  For instructions on generating these files, see:
-# https://download.oracle.com/javase/6/docs/technotes/guides/security/jsse/JSSERefGuide.html#CreateKeystore
+# http://download.oracle.com/javase/6/docs/technotes/guides/security/jsse/JSSERefGuide.html#CreateKeystore
```

Comment:
http -> https

Reply:
I just copied this file from the original distribution with some smaller changes from our side. I don't want to change issues which came from the original. This would make it hard to see difference, if we update cassandra.

Reply:
Ah, didn't realize that. Thanks for letting me know